### DISCUSSION TITLE
Vroid SDK available for JavaScript?

### DISCUSSION THREAD

**Discussion by tejasnayak25** [pixiv/vroid-sdk-developers#62] [Q&A]
I would like to know if Vroid SDK is available for JavaScript. Something like an npm package?

**Comment by nyamadan** [comment]
We don't provide VRoid SDK for JavaScript, only for Unity.

**Comment by tejasnayak25** [comment]
Then is there another way to connect to Vroid Hub through JavaScript?

**Reply by nyamadan** [reply]
I’m afraid we cannot meet your expectation.
Currently, only VRoid SDK can connect to VRoid Hub.
